content/applications/productivity/iot/devices/screen.rst

```diff
@@ -66,27 +69,27 @@ Things)` Box homepage.
 Usage
 =====
 
-Show Point of Sales orders to customers
----------------------------------------
+Show Point of Sale orders to customers
+--------------------------------------
 
-To use the screen in the :guilabel:`Point of Sale` app, go to :menuselection:`Point of Sale -->
-Configuration --> Point of Sale`, open the desired screen, click on :guilabel:`Edit`, and enable the
+To use the screen in the :menuselection:`Point of Sale` app, go to :menuselection:`Point of Sale -->
+Configuration --> Point of Sale`, open the desired screen, :guilabel:`Edit`, and enable the
 :guilabel:`IoT Box Devices` feature.
 
 Now, select :guilabel:`IoT Box` in the drop-down menu and choose the screen in the
-:guilabel:`Customer Display` option. Then, click :guilabel:`Save`.
+:guilabel:`Customer Display` option. Then, :guilabel:`Save` if required.
 
-    .. image:: screen/screen-pos-screen-config.png
-       :align: center
-       :alt: Connect the screen display to the Point of Sale app.
+.. image:: screen/screen-pos-screen-config.png
```

Comment:
In Runbot 14.0, the name of this section is "IoT Box", not "IoT Box Devices", so I think this image is out of date.

Reply:
updated screen shots.